packages/smb-server/test/test.utils.ts

```diff
@@ -0,0 +1,32 @@
+import { DiContainer } from 'peque-di';
+import { EventService } from 'peque-smb-commons';
+
+import { MessageCommand } from '../src/commands';
+import { SubscribeService } from '../src/services';
+
+export const DI = new DiContainer();
+
+const providers = [SubscribeService, EventService, MessageCommand];
+export const loadProviders = (): void => {
+  for (const provider of providers) {
+    DI.set(provider, provider.name);
+  }
+};
+
+async function wait(interval: 'short' | 'medium' | 'long' | number = 'short'): Promise<void> {
```

Comment:
Probably worth making a package with those test utils

Reply:
I will manage this in the next commits.